### ISSUE TITLE
A brief clear title explaining the bug

### ISSUE THREAD

**Issue by richardmoizeau** [openmsupply/open-msupply#1729]
## What went wrong? 😲
 

Discussed with @alainsussol and @JonnaQuismundo
Also tagging @mark-prins    

demo-djibouti

- store Balbala 2
- code: P1ARLU001 
- pref: allow pack to one conversion = true
- pref: convert pack to 1 when receiving = true

<img width="956" alt="Screenshot 2023-05-09 at 10 53 01 PM" src="https://github.com/openmsupply/open-msupply/assets/74992958/b8b96569-6d4d-4320-8a17-57e5ce29cef7">

- store preference: set pack to 1 for all visible items in this store = false (we only this for some items)

<img width="956" alt="Screenshot 2023-05-09 at 10 53 01 PM" src="https://github.com/openmsupply/open-msupply/assets/74992958/da51709a-617e-4f1f-83df-008be1ce59b7">

When receiving an inbound shipment for item P1ARLU001 in Balbala 2, pack size is still "100" instead of "1"

## Expected behaviour 🤔

Pack size should be "1"

## Your environment 🌱

 - Version: 1.1.11
 - Platform:
   - [ ] browser (chrome)

**Comment by DhanyaHerath** [comment]
@richardmoizeau do you need the pack to one preference to work on this fine-grained level for Djibouti?

**Comment by richardmoizeau** [comment]
> @richardmoizeau do you need the pack to one preference to work on this fine-grained level for Djibouti?

Yes for some of their malaria items - may be able to find another solution though - let me know
